webpack.config.js

```diff
@@ -181,6 +183,11 @@ const config = (module.exports = {
   cache: useFilesystemCache
     ? {
         type: "filesystem",
+        cacheDirectory: path.resolve(
+          __dirname,
+          "node_modules/.cache/",
+          edition === "oss" ? "webpack-oss" : "webpack-ee",
```

Comment:
https://webpack.js.org/configuration/cache/#cachecachedirectory
default folder is `node_modules/.cache/webpack`